src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1144,19 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Prefix to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+    	if (fullString.startsWith(prefix)){
+    		return fullString.replaceFirst(prefix, "");
+    	}
+    	else{
```

Comment:
coding standard violation! please check the conventions used to write conditional statements.